test/parallel/test-runner-extraneous-async-activity.js

```diff
@@ -10,7 +10,7 @@ const { spawnSync } = require('child_process');
     fixtures.path('test-runner', 'extraneous_set_immediate_async.mjs'),
   ]);
   const stdout = child.stdout.toString();
-  assert.match(stdout, /^# Warning: Test "extraneous async activity test" generated asynchronous activity after the test ended/m);
+  assert.match(stdout, /^# Warning: Test "extraneous async activity test" at .+extraneous_set_immediate_async\.mjs:3:1 generated asynchronous activity after the test ended/m);
```

Comment:
```suggestion
  assert.match(stdout, /^# Warning: Test "extraneous async activity test" at .+extraneous_set_immediate_async\.mjs:\d+:\d+ generated asynchronous activity after the test ended/m);
```

(same for rest of the file)

Reply:
I am ok with this being a number. it shouldn't change much